File: library/src/main/java/ru/dev/gbixahue/library/android/log/Logger.java
package ru.dev.gbixahue.library.android.log;

/**
 * Created by Anton Zhilenkov on 06.03.2018.
 */
public interface Logger {

	void e(Object from, String msg, Object value);
	void d(Object from, String msg, Object value);
	void setLogHandler(LogHandler logHandler);
}

File: library/src/main/java/ru/dev/gbixahue/library/android/log/Logger.kt
package ru.dev.gbixahue.library.android.log

import ru.dev.gbixahue.library.android.log.profiling.LogProfiler

/**
 * Created by Anton Zhilenkov on 10.10.17.
 */

interface Logger {

	fun d(from: Any, msg: String, value: Any?)
	fun w(from: Any, msg: String, value: Any?)
	fun e(from: Any, msg: String, value: Any?)
	fun setProfiler(profiler: LogProfiler?)
	fun addHandler(handler: LogHandler?)
}
